components/scream/scripts/yaml-to-xml

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
```

Comment:
Was this just to generate an xml from our current `scream_input.yaml` file? Or do you see it being used during case.xyz? If it's the former, do we need to keep it? Once we have our `namelist_defaults_scream.xml` file, we should not need to manually create a yaml file anymore, right? Even less so, generate an xml from a yaml...

Reply:
Yes, we can probably remove this once the PR is ready.